### PR TITLE
Revert "Add instructions for working around NoScript XSS upload problem"

### LINKED ISSUE #4160: Remove NoScript workaround instructions from Source Interface
In #4153, we introduced instructions to the Source Interface for working around a NoScript/Firefox issue which causes some uploads to fail. The upstream NoScript issue (closed as a Firefox issue is being pinpointed as the root cause) is https://github.com/hackademix/noscript/issues/64; the best candidate for the underlying Firefox issue appears to be: https://bugzilla.mozilla.org/show_bug.cgi?id=1506562

These instructions are cumbersome and the upstream issue will hopefully be resolved ASAP. We'll need to continue to monitor upstream and remove the instructions once a fix has landed in the latest releases of Tor Browser Bundle & Tails.

### PR DESCRIPTION
**Pull request by eloquence**
Resolves #4160. 

This reverts commit b35530831db8bcb191f2c8553b1f05fa7e6eab6d (and the dependent commit babc2f52da21c0b8d15eb4bef3c1b6c66b37e744).

## Status

Ready for review

## Test plan

1. Build updated staging environment
2. Connect to source interface using Tor browser version 8.0.8 or greater
3. Verify that "Scan uploads for potential cross-site requests" option (under "Advanced" NoScript settings) is not set. It should be unset by default.
3. Verify that NoScript instructions implemented in #4160 no longer appear on source interface
4. Upload a test file of at least 75MB size repeatedly (I recommend a total of 5 times)

Expected result:
- All uploads complete successfully.

## Deployment

No special considerations. Tor browser includes the required fix as of [version 8.0.8](https://blog.torproject.org/new-release-tor-browser-808), which was also [shipped with Tails 3.13.1](https://tails.boum.org/news/version_3.13.1/index.en.html), and which should auto-updated on most systems.

## Checklist

- [x] Linting (`make ci-lint`) and tests (`make -C securedrop test`) pass in the development container

### CODE REVIEW

File: securedrop/sass/_base.sass [hunk 1]
```diff
@@ -71,10 +71,6 @@
 @import "modules/grid-item"
 // Option - might be unused. Delete?
 @import "modules/option"
-// Flexbox - support for flexbox layouts
-@import "modules/flexbox"
```
Comment (rmol) [inline securedrop/sass/_base.sass]: The flexbox module is used elsewhere, so needs to be kept. Not sure how it got swept up in the revert, as it was added in https://github.com/freedomofpress/securedrop/commit/4ae0c3f3fded39f5eb6837bb7ccc74a29bd003e3#diff-e95835e98013102be1f7b1c88d5505e0.

**Comment by rmol** [comment]
I've tested this by uploading a 100MB file five times with no failures. The only problem I see is that the revert somehow picked up the flexbox SASS module, which is used in other places.

File: securedrop/sass/_base.sass (hunk 1 above)
Comment (eloquence) [reply securedrop/sass/_base.sass]: If I remember correctly, the revert triggered a conflict; I must have accidentally removed it on resolution. Will amend.

**Comment by rmol** [comment]
Looks good. I checked an instance of content that was misaligned without the flexbox module and it's fine now.

**Pull request merged**